.github/workflows/ci_server.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests when there is a pull request
+# with changes within the "server" directory 
+
+name: CI_SERVER
+
+on:
+  pull_request:
+    branches: 
+    - master 
+    paths: 
+    - 'server/**'
+    - '!server/README.md'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest   
+
+    strategy:
+      matrix:
+        node-version: [10.x]
```

Comment:
I suspect that NodeJS v10 will cause some issues with #34 . I couldn't get Jest to run in Node v10, but I can get it to work in Node v12 and v13. Running the server's test scripts in Node v11 generates some deprecation warnings.

Is Node v12 or v13 available for us to use in this workflow?

Reply:
Not too sure about Node v13 but I know that that anode v12 is available, can change it.

Reply:
That'll be great. Might also want to change the version for the frontend for consistency - I'm guessing we'll probably be wanting to run Jest on the frontend too.